suricata/update/main.py

```diff
@@ -687,13 +687,23 @@ def build_rule_map(rules):
     number will be used.
     """
     rulemap = {}
-
+    
     for rule in rules:
         if rule.id not in rulemap:
             rulemap[rule.id] = rule
+            FORMAT = '%(version)s %(message)s'
+            logging.basicConfig(format=FORMAT)
+            d={'version: rule'}
+            logger = logging.getLogger('tcpserver')
```

Comment:
You do not need to configure logger multiple times, just loop up how it has been used in the rest of the code. Now. could you please write down the steps you're using for testing? Make sure to write the steos down after having tried them on your system.

Reply:
In addition to this, I would also request you to have a look at the Travis logs and understand what have you done incorrectly (logically).